Readme.md

```diff
@@ -2,9 +2,12 @@
 
 -----
 
-## This is the documentation for the unreleased version 2.0
+## This is the documentation for version 2.0.0-alpha
 
-**For the current version 1.x documentation, see [the v1.x branch](https://github.com/Automattic/node-canvas/tree/v1.x)**
+**For version 1.x documentation, see [the v1.x branch](https://github.com/Automattic/node-canvas/tree/v1.x)**
+
+Alpha versions of 2.0 can be installed using `npm install canvas@v2.0.0-alpha.5`,
```

Comment:
How about pointing people to run `npm install canvas@next`, that will automatically install the latest alpha, so no need to lookup the number

Reply:
I always forget about that pseudotag... done.